### DISCUSSION TITLE
✅ Disable/Deactivate Widget in Editor View

### DISCUSSION THREAD

**Discussion by acarravedo** [elementor/elementor#24279] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I can't temporarily disable a widget in the edit view so that it doesn't load, the only option is to make it invisible in the different breakpoints, but this makes the widget always load.

### Describe the solution you'd like

I want to be able to temporarily disable a widget on any page so that it is not shown in the front view, this way I can have widgets mounted that do not load.

A "simple check" that allows me to do this, in the style of the "breakpoints visibility" would be sufficient.

**Comment by nicholaszein** [comment]
Hello. This functionality already exists since version 2.2 with the introduction of the Structure Panel / Navigator. All you need to do is to click the eye icon (👁️) to temporarily show or hide an element in the Editor Preview.

You can learn more in the following article in the Help Center: https://elementor.com/help/navigator/#hide-show-elements

Kind regards
